fastlane/Fastfile

```diff
@@ -431,12 +480,36 @@ end
     end
 
     promo_screenshots(
-      orig_folder: screenshots_directory,
+      orig_folder: options[:source] || screenshots_directory,
       metadata_folder: File.join(Dir.pwd, "metadata"),
       output_folder: File.join(Dir.pwd, "/promo_screenshots"),
       force: options[:force] || true
     )
   end
+
+  desc "Rebuild Screenshot Devices"
+  lane :rebuild_screenshot_devices do |options|
+    require 'simctl'
+
+    # Using flatten here because we may be getting a comma separated string if
+    # called from the command line via "fastlane run" or an array if called by
+    # another action that has already preformatted the value into an array.
+    device_names = (options[:devices] || screenshot_devices).split(',').flatten
```

Comment:
Is this code readable? I like how it's a one liner, but maybe I could split it into something easier to read, or extract it in a function?